chipy_org/templates/homepage.html

```diff
@@ -230,7 +232,7 @@ <h3>This Month's Topics</h3>
                       {% if topic.experience_level %}
                       Experience Level: {{ topic.get_experience_level_display }}<br />
                       {% endif %}
-                      {{ topic.description|striptags|truncatewords:80|safe }}<br />
+                      {{ topic.description|striptags|truncatewords:80|bleach|safe }}<br />
```

Comment:
A minor enhancement is to remove the striptags filter and use truncatewords_html.  This leaves any formatting from allowed tags intact.

Reply:
Nice, I believe we were looking to have the homepage stripped down and the details page to have the rich text.

Reply:
Thanks Aaron!!😀